### PR TITLE
WAITTENSION, reduce usage of # type: ignore

### PR DESCRIPTION
**Pull request by pkubanek**

### CODE REVIEW

File: python/lsst/ts/criopy/gui/TimeChart.py [hunk 1]
```diff
@@ -232,16 +232,18 @@ def _topicSelected(self, obj: UnitLabel) -> None:
                     continue
 
                 if self._signal is not None:
-                    self._signal.disconnect(self._appendData)  # type: ignore
+                    self._signal.disconnect(self._appendData)  # t
```
Comment (teweitsai) [inline python/lsst/ts/criopy/gui/TimeChart.py:235]: You do not need "# t" now.

File: python/lsst/ts/criopy/m1m3/force_actuator/HistogramPageWidget.py [hunk 2]
```diff
@@ -94,7 +94,7 @@ def updateValues(self, data: typing.Any) -> None:
         data : `object`
             New data structure, passed from SAL handler.
         """
-        if data is None:
+        if data is None or self.field is None:
```
Comment (teweitsai) [inline python/lsst/ts/criopy/m1m3/force_actuator/HistogramPageWidget.py:97]: I thought `black` will do?

```
if (data is None) or (self.field is None):
```

**Review by teweitsai — APPROVED** [review]
Looks good.

File: python/lsst/ts/criopy/gui/TimeChart.py (hunk 1 above)
Comment (pkubanek) [reply python/lsst/ts/criopy/gui/TimeChart.py:235]: Good catch, removed

File: python/lsst/ts/criopy/m1m3/force_actuator/HistogramPageWidget.py (hunk 2 above)
Comment (pkubanek) [reply python/lsst/ts/criopy/m1m3/force_actuator/HistogramPageWidget.py:97]: No. It passes pre-commit, black doesn't do that. Even when running black on the file, this isn't changed.

**Pull request merged**